mead/utils.py

```diff
@@ -16,6 +17,9 @@
 logger = logging.getLogger('mead')
 
 
+KNOWN_FMTS = ['%Y%m%d', '%Y-%m-%d', '%Y/%m/%d', '%Y', '%Y%m', '%Y-%m', '%Y/%m', '%Y%m%d_%H%M', '%Y-%m-%d_%H-%M', '%Y/%m/%d_%M/%H']
```

Comment:
not very descriptive: `KNOWN_DATE_FORMATS`?

Reply:
Fixed